### DISCUSSION TITLE
新版视频页评论区（大概在前两天）更新了

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3425] [综合讨论 / General]
闲的没事点了“前往新版”按钮

然后发现大改了一通，估摸着V2的简化评论区和复制评论区链接这两个功能和其他一些评论区相关的功能应该彻底废了，https://github.com/the1812/Bilibili-Evolved/blob/master/src/components/utils/comment-apis.ts 这个api可能也需要更新更新
</br>
V1——
</br>
顺便吐槽一下这个注释（
![图片](https://user-images.githubusercontent.com/81268244/172928312-ae96b933-5a4e-4b15-8342-4781b0bd20c0.png)

**Comment by MC-dusk** [comment]
现在还没修好吗🙏

**Reply by the1812** [reply]
挺早就修了, 最近刚阳完回来, 元旦左右会发一下
